packages/documentation-framework/scripts/webpack/webpack.base.config.js

```diff
@@ -55,20 +55,12 @@ module.exports = (_env, argv) => {
             options: {
               cacheDirectory: '.cache/babel',
               cacheCompression: false,
-              presets: [['@babel/preset-env', {
-                loose: true,
-                corejs: 3,
-                useBuiltIns: 'entry',
-                exclude: ['transform-regenerator', 'transform-async-to-generator'],
-                modules: false,
```

Comment:
I've removed the `modules` field here, as this should be left to `auto` (the default) when using it in combination with a bundler such as WebPack + `babel-loader`, which is the case here ([see documentation](https://babeljs.io/docs/babel-preset-env#modules-auto)).